### ISSUE TITLE
Alt attribute

### ISSUE THREAD

**Issue by jim-rosales** [sskagg/prj-rev-bwfs-dasmoto#2]
When adding images, it is best practice to include the [alt attribute](https://en.wikipedia.org/wiki/Alt_attribute) in every image. The alt attribute is used for both [accessibility](https://www.codecademy.com/articles/accessibility) and as a backup, in case the image cannot be rendered. 

https://github.com/sskagg/prj-rev-bwfs-dasmoto/blob/4c67f98f1e31570012890bd1447ed0b5d28026f1/Dasmoto/index.html#L13
